### ISSUE TITLE
Duplicate sort fields in discovery.xml

### ISSUE THREAD

**Issue by alanorth** [DSpace/DSpace#9104]
**Describe the bug**
DSpace `discovery.xml` defines several duplicate sort fields:

- `sortTitle`: https://github.com/DSpace/DSpace/blob/main/dspace/config/spring/api/discovery.xml#L2856
- `sortTitleAsc`: https://github.com/DSpace/DSpace/blob/main/dspace/config/spring/api/discovery.xml#L2931

Both are _ascending_, which makes me wonder if the first one was supposed to be _descending_?

- `sortDateIssued`: https://github.com/DSpace/DSpace/blob/main/dspace/config/spring/api/discovery.xml#L2861
- `sortDateIssuedDesc`: https://github.com/DSpace/DSpace/blob/main/dspace/config/spring/api/discovery.xml#L2936

Both are _descending_, which makes me wonder if the first one was supposed to be _ascending_?

**Expected behavior**
This is not a bug, but is incorrect configuration and can lead to confusion and incorrect sorting for site administrators.

**Comment by alanorth** [comment]
I just noticed this issue again. Is there any chance we can settle on a convention for naming sort fields? We currently have a mix of default sort orders for our sort fields, which is confusing and leads to bugs like this one.

The easy fix is to adjust the names of the duplicate sort fields.

A better fix is to make the order of all sort fields explicit, ie `sortTitleAsc` and `sortTitleDesc` so there is less ambiguity.

**Comment by kshepherd** [comment]
I agree, making it explicit for all sort fields and removing the old names that don't include sort direction, would make configuration clearer.
